### ISSUE TITLE
Implement a new mode "massacre"

### ISSUE THREAD

**Issue by vkolotov** [Docroma47/chess#9]
A new chess demo mode is to be implemented which is called "massacre".

This mode is when a chess piece walks on the board and eats the other enemy pieces. Enemy pieces are the pieces of different colour. Allies pieces are pieces of the same colour.

This new mode has only one argument and can be executed as per this:

```bash
java -classpath target/chess-1.0-SNAPSHOT-jar-with-dependencies.jar org.mom47.chess.App massacre 15
```
The argument is the initial number of chess pieces on the chess board. 
Chess pieces are chosen **randomly** from all 32 pieces and set on **random** squires.

After the chess pieces are set on the board, a chess piece is chosen **randomly** of the pieces on the board. This piece becomes an **active** piece which should walk on the board randomly as much as in the "snake" mode and "eat" enemy pieces. 

When the active piece riches chess boarder it changes its direction randomly (like in the "snake" mode). 

When it steps onto another pice of different colour (the enemy) - it eats it (the enemy piece is removed from the board). The active piece takes "eaten" piece squire and continues to walk on the board (e.g. it does not change its direction when it eats another chess piece). 

When the active piece comes across its allies, then it changes its direction randomly (like in the "snake" mode).

In short: the active piece eats enemy pieces and changes direction if comes across borders or allies pieces.
